### DISCUSSION TITLE
Channels that stream X content over a certain period [Night time]

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#111] [Q&A] [answered]
How shall we treat these kind of streams ?

Channels that are onto this type of streams contain Cinema+ (CO) and Rouge TV (CH).

**Comment by benevenstanciano** [comment, marked as answer]
I'd suggest it be flagged as adult since there's not much you can do about time zones. Someone in Quebec cycling through to Rouge TV in the afternoon would see adult content.

**Reply by BellezaEmporium** [reply]
I agree.
